index.html

```diff
@@ -13359,66 +13368,30 @@ <h3>States and Properties</h3>
 					<td><sref>aria-checked</sref></td>
 					<td><code>false</code> </td>
 				</tr>
-				<tr>
-					<td><rref>switch</rref></td>
-					<td><sref>aria-checked</sref></td>
-					<td><code>false</code> </td>
-				</tr>
-				<tr>
-					<td><rref>combobox</rref></td>
-					<td><sref>aria-expanded</sref></td>
-					<td><code>false</code></td>
-				</tr>
-				<tr>
-					<td><rref>combobox</rref></td>
-					<td><sref>aria-haspopup</sref></td>
-					<td><code>listbox</code></td>
-				</tr>
-
-				<tr>
-					<td><rref>heading</rref></td>
-					<td><pref>aria-level</pref></td>
-					<td><code>2</code></td>
-				</tr>
-				<tr>
-					<td><rref>separator</rref> (if focusable)</td>
-					<td><pref>aria-valuenow</pref></td>
-					<td><code>(aria-valuemax - aria-valuemin) / 2</code></td>
-				</tr>
 				<tr>
 					<td><rref>scrollbar</rref></td>
 					<td><pref>aria-controls</pref></td>
 					<td>no mapping</td>
 				</tr>
 				<tr>
 					<td><rref>scrollbar</rref></td>
-					<td><pref>aria-orientation</pref></td>
-					<td><code>vertical</code></td>
+					<td><pref>aria-valuenow</pref></td>
+					<td>If missing or not a <a href="#valuetype_number">number</a>,<code>(aria-valuemax - aria-valuemin) / 2</code>. If present but less than <code>aria-valuemin</code>, the value of <code>aria-valuemin</code>. If present but greater than <code>aria-valuemax</code>, the value of <code>aria-valuemax</code>.</td>
 				</tr>
 				<tr>
-					<td><rref>scrollbar</rref></td>
+					<td><rref>separator</rref> (if focusable)</td>
 					<td><pref>aria-valuenow</pref></td>
-					<td>If missing or not a <a href="#valuetype_number">number</a>,<code>(aria-valuemax - aria-valuemin) / 2</code>. If present but less than <code>aria-valuemin</code>, the value of <code>aria-valuemin</code>. If present but greater than <code>aria-valuemax</code>, the value of <code>aria-valuemax</code>.</td>
+					<td><code>(aria-valuemax - aria-valuemin) / 2</code></td>
```

Comment:
Why isn't this the same as for scrollbar and slider?

Reply:
Good question. That's how it was before: https://w3c.github.io/aria/#authorErrorDefaultValuesTable

Now that the table is sorted so that separator is between scrollbar and slider, that difference seems more obvious.

I see no reason why a focusable separator wouldn't have the same fallback conditions for aria-valuenow as scrollbar and slider, so I'll change it to match.